src/loc/pt-br.ts

```diff
@@ -0,0 +1,358 @@
+    UnderlineTitle: "Sublinhado (Ctrl + U)",
+    AlignTitle: "Alinhar",
+    ListTitle: "Lista",
+    LinkTitle: "Hyperlink",
+    ImageTitle: "Imagem",
+    MoreTitle: "Mais",
+    FormattingPaneTitle: "Formatação de texto",
+    CloseButton: "Fechar",
+    InsertLinkTitle: "Inserir link",
+    InsertImageTitle: "Inserir Imagem",
+    AddressFieldLabel: "Endereço",
+    TextToDisplayLabel: "Texto a ser exibido",
+    SaveButtonLabel: "Salvar",
+    CancelButtonLabel: "Cancelar",
+    RemoveLinkLabel: "Remover link",
+    ParagraphSectionTitle: "Parágrafo",
+    HyperlinkSectionTitle: "Hyperlink",
+    UndoTitle: "Desfazer (Ctrl + Z)",
+    RedoTitle: "Refazer (Ctrl + Y)",
+    ClearFormattingTitle: "Limpar toda a formatação",
+    FontStyleTitle: "Estilo de fonte",
+    FontSizeTitle: "Tamanho da fonte",
+    StrikethroughTitle: "Tachado",
+    SuperscriptTitle: "Sobrescrito",
+    SubscriptTitle: "Subscrito",
+    FontColorLabel: "Cor da fonte",
+    AutomaticFontColor: "Automático",
+    HighlightColorLabel: "Cor de destaque",
+    NoColorHighlightColor: "Sem cor",
+    IncreaseIndentTitle: "Aumentar recuo",
+    DecreaseIndentTitle: "Recuar devagar",
+    ThemeColorsGroupName: "Cores do tema",
+    HighlightColorsGroupName: "Destacar cores",
+    StandardColorsGroupName: "Cores padrão",
+    CustomColorsGroupName: "Cores personalizadas",
+    ThemeColorDarker: "Tema mais escuro",
+    ThemeColorDark: "Tema escuro",
+    ThemeColorDarkAlt: "Tema escuro alternativo",
+    ThemeColorPrimary: "Tema principal",
+    ThemeColorSecondary: "Tema secundário",
+    ThemeColorTertiary: "Terciário neutro",
+    ThemeColorNeutralSecondary: "Neutro secundário",
+    ThemeColorNeutralPrimaryAlt: "Alternativo primário neutro",
+    ThemeColorNeutralPrimary: "Neutro primário",
+    ThemeColorNeutralDark: "Neutro escuro",
+    HighlightColorYellow: "Amarelo",
+    HighlightColorGreen: "Verde",
+    HighlightColorAqua: "Aqua",
+    HighlightColorMagenta: "Magenta",
+    HighlightColorBlue: "Azul",
+    HighlightColorRed: "Vermelho",
+    HighlightColorDarkblue: "Azul escuro",
+    HighlightColorTeal: "Cerceta",
+    HighlightColorDarkgreen: "Verde escuro",
+    HighlightColorPurple: "Roxo",
+    HighlightColorMaroon: "Marrom",
+    HighlightColorGold: "Ouro",
+    HighlightColorDarkgrey: "Cinza escuro",
+    HighlightColorGrey: "Cinza",
+    HighlightColorBlack: "Preto",
+    StandardColorDarkred: "Vermelho escuro",
+    StandardColorRed: "Vermelho",
+    StandardColorOrange: "Laranja",
+    StandardColorYellow: "Amarelo",
+    StandardColorLightgreen: "Luz verde",
+    StandardColorGreen: "Verde",
+    StandardColorLightblue: "Azul claro",
+    StandardColorBlue: "Azul",
+    StandardColorDarkblue: "Azul escuro",
+    StandardColorPurple: "Roxo",
+
+    DatePickerMonthLongJanuary: "Janeiro",
+    DatePickerMonthShortJanuary: "Jan",
+    DatePickerMonthLongFebruary: "Fevereiro",
+    DatePickerMonthShortFebruary: "Fevereiro",
```

Comment:
@joelfmrodrigues DatePickerMonthShort should be short "Fev" but appear long, the same until December.